docs/tutorials/where_do_I_get_papers.md

```diff
@@ -0,0 +1,125 @@
+# Where to get papers
+
+## OpenReview
+
+You can use papers from [https://openreview.net/](https://openreview.net/) as your database!
+Here's a helper that fetches all papers from a selected conference (like ICLR, ICML, NeurIPS) and downloads relevant papers using LLM. Install `openreview-py` with
+
+```bash
+pip install paper-qa[openreview]
+```
+
+and get your username and password from the website. You can put them into `.env` file under `OPENREVIEW_USERNAME` and `OPENREVIEW_PASSWORD` variables, or pass them in the code directly.
+
+```python
+from paperqa import Settings
+
+# these settings require gemini api key you can get from https://aistudio.google.com/
+# import os; os.environ["GEMINI_API_KEY"] = os.getenv("GEMINI_API_KEY")
+# 1Mil context window helps to suggest papers
+settings = Settings.from_name("openreview")
```

Comment:
Maybe make a comment here on if these settings are required to get the papers from OpenReview